gravitee-plugin-core/src/main/java/io/gravitee/plugin/core/internal/PluginEventListener.java

```diff
@@ -157,6 +158,7 @@ protected void doStart() throws Exception {
         eventManager.subscribeForEvents(this, PluginEvent.class);
     }
 
+    @VisibleForTesting
```

Comment:
method being public, why do you need this annotation?

Reply:
It's public because the test is not in the correct package, i'll move the test and reduce visibility